projects/storefrontlib/src/cms-components/navigation/breadcrumb/breadcrumb.component.html

```diff
@@ -1,6 +1,12 @@
-<nav>
-  <span *ngFor="let crumb of crumbs$ | async">
-    <a [routerLink]="crumb.link" [innerHTML]="crumb.label"></a>
-  </span>
+<nav aria-label="Breadcrumb">
+  <ol>
+    <li span *ngFor="let crumb of crumbs$ | async; last as isLast">
+      <a
+        [routerLink]="crumb.link"
+        [innerHTML]="crumb.label"
+        [attr.aria-current]="isLast ? 'page' : null"
```

Comment:
string 'page' also looks as hardcoded. Let's add `aria-current` to list in our `i18n-lint` command